lib/ContributorDetails/ContributorDetailsContainer.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Spinner } from '@folio/stripes/components';
+
+import { contributorNameTypesManifest } from '../manifests';
+import ContributorDetails from './ContributorDetails';
+
+const ContributorDetailsContainer = ({ resources, contributors }) => {
+  const contributorNameTypes = get(resources, 'contributorNameTypes.records');
+
+  if (!contributorNameTypes) {
```

Comment:
basically this doesn't work right with stripes resources. It will just add a blink at the beginning of fetch. Right after that it will be an empty array. So it makes sense only with `fetch: false`, when you won't have an empty array until fetching is finished

Reply:
Removed it